src/diff/props.js

```diff
@@ -104,20 +106,22 @@ export function setProperty(dom, name, value, oldValue, namespace) {
 			// - className --> class
 			name = name.replace(/xlink(H|:h)/, 'h').replace(/sName$/, 's');
 		} else if (
-			name != 'width' &&
-			name != 'height' &&
-			name != 'href' &&
-			name != 'list' &&
-			name != 'form' &&
-			// Default value in browsers is `-1` and an empty string is
-			// cast to `0` instead
-			name != 'tabIndex' &&
-			name != 'download' &&
-			name != 'rowSpan' &&
-			name != 'colSpan' &&
-			name != 'role' &&
-			name != 'popover' &&
-			name in dom
+			split[0] == 'prop' ||
+			(split[0] != 'attr' &&
```

Comment:
Not sure if it will make a difference: Usually it's faster to have the common case upfront. The `prop` prefix being there is probably not the common case.

Reply:
Very true.

This was inserted quick just so I could check the perf & side, truth be told, but will swap it to the bottom.